server/controllers/templates/web_templates.go

```diff
@@ -400,32 +400,41 @@ var ProjectJobsTemplate = template.Must(template.New("blank.html.tmpl").Parse(`
     <style>
       #terminal {
         position: fixed;
-        top: 200px;
+        top: 0px;
         left: 0px;
         bottom: 0px;
         right: 0px;
         border: 5px solid white;
+        z-index: 10;
         }
 
       .terminal.xterm {
         padding: 10px;
       }
+      #watermark {
+        opacity: 0.5;
+        color: BLACK;
+        position: absolute;
+        bottom: 0;
+        padding-right: 30px;
+        padding-bottom: 15px;
+        right: 0;
+        z-index: 15;
+      }
     </style>
   </head>
 
   <body>
-    <section class="header">
+    <section id="watermark">
     <a title="atlantis" href="{{ .CleanedBasePath }}/"><img class="hero" src="{{ .CleanedBasePath }}/static/images/atlantis-icon_512.png"/></a>
-    <p class="title-heading">atlantis</p>
-    <p class="title-heading"><strong></strong></p>
+    <p class="title-heading-white">atlantis</p>
```

Comment:
This isn't really a 'title heading' anymore, so how about a different name?

Reply:
@X-Guardian it's now called "terminal-heading-white". I believe you can now merge this PR.